tests/platform_helper/test_command_environment.py

```diff
@@ -831,6 +833,54 @@ def test_when_https_listener_present(self, find_load_balancer):
         assert "listener_arn" == listener_arn
 
 
+class TestFindHTTPSCertificate:
+    @patch(
+        "dbt_platform_helper.commands.environment.find_https_listener",
+        return_value="https_listener_arn",
+    )
+    def test_when_no_certificate_present(self, find_https_certificate):
+        from dbt_platform_helper.commands.environment import CertificateNotFoundError
+        from dbt_platform_helper.commands.environment import find_https_certificate
+
+        boto_mock = MagicMock()
+        boto_mock.client().describe_listener_certificates.return_value = {"Certificates": []}
+        with pytest.raises(CertificateNotFoundError):
```

Comment:
Just to help distinguish the arrange act assert parts...
```suggestion

        with pytest.raises(CertificateNotFoundError):
```